sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -20,7 +20,7 @@ WITH stripe_subscriptions AS (
     cancel_at,
     cancel_at_period_end,
     ended_at,
-    COALESCE(ended_at, CURRENT_TIMESTAMP) AS end_date,
+    COALESCE(ended_at, TIMESTAMP(CURRENT_DATE)) AS end_date,
```

Comment:
This was causing `months_retained` to potentially exceed `current_months_since_subscription_start`, which it should not do.